sdk/helper/kdf/counter_impl.go

```diff
@@ -0,0 +1,100 @@
+// See note about this package in kdf.go; these implementations aim to provide
+// a PKCS#11 v3.0 CKM_SP800_108_*_KDF compatible interface.
+package kdf
+
+import (
+	"fmt"
+	"hash"
+	"io"
+)
+
+type counterKDF struct {
+	// Globally provisioned values
+	prf        hash.Hash
+	parameters []KBKDFParameter
+	keyLengths []int
+
+	// Internal counter for this KDF instance.
+	counter uint64
```

Comment:
`go fmt ./*.go` from `sdk/helper/kdf` results in no changes to these files (Go 1.17.2). 

Because of the comments between members, `go fmt` won't align the types. If you remove the comments, it will reformat them... Should I remove the comments?

Reply:
No, that's fine.  Plenty of precedent for that.  Was just making sure that source was getting auto-formatted.